modules/ROOT/pages/setup.adoc

```diff
@@ -58,7 +62,7 @@ All B2B processing Mule apps are deployed to this server group. The steps to set
 IMPORTANT: The server group you setup must use the exact name: *B2BServerGroup*. This name is case sensitive.
 +
 
-. For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Downloading the MuleSoft runtime] and unzipping it.
+. For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Download and Install Mule] and unzipping it.
```

Comment:
```suggestion
. For each environment, set up your Mule instance by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[downloading and installing Mule] and unzipping it.
```

Reply:
Fixed. The second one is  the title of the doc, so won't change.